website/src/components/webinars/WebinarLayout.tsx

```diff
@@ -0,0 +1,120 @@
+import { useChiliPiper } from '../../hooks/chiliPiper'
+
+interface Props {
+    customer?: Customer
+    title: string
+    subtitle: string
+    description: React.ReactNode
+    speakers: Speaker[]
+    children?: React.ReactNode
+}
+
+interface Customer {
+    name: string
+    logo: string
+    href: string
+}
+
+interface Speaker {
+    name: string
+    title: string
+    img: string
+    bio: string
+}
+
+export const WebinarLayout: React.FunctionComponent<Props> = ({
+    title,
+    subtitle,
+    customer,
+    description,
+    speakers,
+    children,
+}) => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: '66361163-5e08-4be3-8ab0-6590b70df69e',
+        targetId: 'form',
+    })
+    useChiliPiper()
+
+    const windowWidth = useWindowWidth()
+    const isMdScreen = windowWidth < breakpoints.lg && windowWidth > breakpoints.sm
+    const isMobile = windowWidth < breakpoints.md
+
+    return (
+        <>
+            {/* Hero style differs if webinar is customer- vs. interal-based */}
+            <section className={classNames(!customer && styles.hero)}>
+                <div className="container py-6 d-flex flex-column flex-lg-row justify-content-around align-items-center">
+                    {customer && (
+                        <div className="col-lg-6 col-12 text-lg-center pb-5 pb-lg-0">
+                            <img
+                                className={`border-right border-black ${
+                                    isMobile || isMdScreen ? 'border-2 mr-3 pr-3' : 'border-3 mr-4 pr-4'
+                                }`}
+                                width={isMobile ? '65' : isMdScreen ? '80' : '110'}
+                                src="/sourcegraph/sourcegraph-mark.svg"
+                                alt="Sourcegraph mark"
+                            />
+                            <img
+                                height={isMobile ? '15' : isMdScreen ? '20' : '25'}
+                                src={customer.logo}
+                                alt={`${customer.name} logo`}
+                            />
+                        </div>
+                    )}
+
+                    <div className={classNames('col-12', customer ? 'col-lg-6' : 'text-white')}>
+                        <h2 className="display-3 font-weight-bold mb-4">{title}</h2>
+                        <h3 className={classNames('font-weight-light', { 'max-w-400': customer })}>{subtitle}</h3>
+                    </div>
+                </div>
+            </section>
+
+            <section className="bg-white py-6">
+                <ContentSection className="d-flex flex-column-reverse flex-md-row">
+                    {description}
+
+                    <div className="col-md-6 col-12 pb-md-0 pb-6">
+                        <h3 className="font-weight-bold">Watch the on-demand webinar</h3>
+                        <div className="border border-3 border-plum-mist pt-4 px-4 pb-2 mx-1 mt-3">
```

Comment:
It looks like the Figma uses a color gradient on the border in the desktop view, and a solid color on the mobile view. Perhaps we could make a note in the description that elected to use `border-plum-mist` on both scales. Otherwise, if we choose to implement this as a `linear-gradient`, we could conform these hex values to our brand colors as `plum-mist` and `blue-mist`.

<img width="400" alt="Screen Shot 2022-05-11 at 4 21 51 PM" src="https://user-images.githubusercontent.com/60713139/167949973-65d4ffad-8f91-4171-bb01-a296074779bb.png">

<img width="224" alt="Screen Shot 2022-05-11 at 4 23 11 PM" src="https://user-images.githubusercontent.com/60713139/167950057-28455700-81c8-4e49-a722-4484b753a91b.png">

Reply:
Good eye, I didn't even notice that! Hm, my vote would be to keep a solid color for the border to limit complexity. Because `linear-gradient`'s can only be used as backgrounds, right? It would take some re-working to add in that style to this form. What do you think @bretthayes ? I'll update the PR description with the final resolution here

Reply:
Welp, I guess since you've already got a scss module, you could add this:

```scss
border-image-slice: 1;
border-image-source: linear-gradient(to right, #E2D4F9, #DCEFFB);
```

There's pretty decent [modern browser coverage](https://caniuse.com/?search=border-image) for it.

Funny that those colours aren't even in our brand guidelines. 😅  I guess `violet-mist` to `plum-mist` or `violet-mist` to `blue-mist` would be our closest options available to us if we don't want to introduce some random hex's. 😬 Feel free to do either option in the interim cause we'll likely revisit this in our Tailwind conversions after we've consolidated our colours with Tracey, and after our designers follow those consolidated colours for future designs.

cc: @traceyljohnson

Reply:
TIL. That's so cool! It's not wanting to apply to the element, though. If you have any ideas what's missing, I'm all ears. I was troubleshooting for ~45 min to no avail. I'll troubleshoot some more on Monday

![image](https://user-images.githubusercontent.com/59381432/168181702-39a6dd55-ecff-4e0c-98f5-95cba1f5b4e6.png)

Reply:
Adding `!important` for the rules fixes it

Reply:
This is looking so great. Great to get this info on the implementation, too! Thank you!